src/test/java/StringTest.java

```diff
@@ -0,0 +1,54 @@
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class StringTest {
+    @Test
+    @DisplayName("','를 이용하여 String을 split 한다.")
+    void test_requirement_1_1() {
+        String input = "1,2";
+
+        String[] result = input.split(",");
+        assertThat(result).contains("1", "2");
```

Comment:
```suggestion
        assertThat("1,2".split(",")).contains("1", "2");
```
로컬 변수 없이 구현해도 되지 않을까?